docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -25,6 +25,17 @@ Yes. Customers can subscribe to Teleport Enterprise Cloud updates at [status.tel
 
 Please [reach out to sales](https://goteleport.com/signup/enterprise) to discuss pricing.
 
+## Are maintenance update times configurable? 
```

Comment:
We're referring to the same thing with different verbiage a few places here.

- maintenance update times
- scheduled maintenance windows
- maintenance start window time
- maintenance time window

Is there any simplification we could do here?